### DISCUSSION TITLE
I dont get it.. where do i configure this?

### DISCUSSION THREAD

**Discussion by best-74** [krahabb/meross_lan#147] [General]
I have installed the Meross Lan repo through HACS... But how or where do i configure it? I have Meross Cloud installed in integrations, and would like to change to this instead.

**Comment by best-74** [comment]
Hmm, got my install cleaned up a bit.. and reinstalled meross_lan. Is it supposed to find my meross switches automaticly via DHCP discovery, or should i manually add them.. I'm running in an docker container (with network mode host)..

**Reply by best-74** [reply]
Well. for others "not getting it".. I got mine up and running by adding static dhcp through my router to my Meross plugs, and then just typing in the ip, for each device.. Works great!

Many thanks for this great integration!!!

**Reply by krahabb** [reply]
Hello @best-74 ,
Sorry for your issues here. I too have a development instance of HA on docker and really can't get it to work with dhcp. I'm not too sure but when I tried to dig into it a bit I remember 'host' mode was the correct way to have dhcp forwarded to the container without too much configuration trouble but..I don't know for sure.
